local-helm/local-overrides.yaml

```diff
@@ -46,6 +46,8 @@ mp:
 
 pb: 
   enabled: true  
+  featureFlags:
```

Comment:
can you please either undo this change, or bring in all the feature flags so that it is consistent?